build-data/paper.at

```diff
@@ -289,3 +289,7 @@ public net.minecraft.world.level.levelgen.SurfaceSystem getOrCreateRandomFactory
 
 # Fix removing recipes
 public net.minecraft.world.item.crafting.RecipeManager byName
+
+# Add Schooling Fish API
```

Comment:
Shouldn't this be changed to Schoolable too? Also, the patch name in general could be renamed to Schoolable.

Reply:
Agreed, will do.

Reply:
Done